src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,36 @@
 package core.basesyntax;
 
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+
 public class WorkWithFile {
     public void getStatistic(String fromFileName, String toFileName) {
-
+        StringBuilder result = new StringBuilder();
+        int supplyCount = 0;
+        int buyCount = 0;
+        try {
+            String fileContent = Files.readString(Path.of(fromFileName));
+            //I was trying to split with System.lineSeparator() but it didn't work by some reason
+            String[] fileSeparatedLines = fileContent.split("\\r?\\n");
```

Comment:
What happened with System.lineSeparator?

Reply:
This is what I have if using System.lineSeparator instead of "\\r?\\n"
![зображення_2023-11-15_213733413](https://github.com/mate-academy/jv-work-with-file/assets/110512851/21d3fce4-fa4b-44ca-82cf-3a374d4fa778)

Reply:
I guess it's not splitting at all, then works next split(","), and crashes after trying to cast String(that suposed to be number if both split works correct) into Integer